src/components/styled-components/StyledInput.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+const StyledInput = styled.input`
+	${({ variant }) =>
+		variant === 'full' &&
+		`
+        width: 30.3rem;
+        height: 4.1rem;
+        border-radius: 1.5rem;
+        border: 0.3rem solid var(--input-border);
+        margin: 0 2.1rem 1rem 2.1rem;
+    `}
+
+	${({ variant }) =>
+		variant === 'date' &&
+		`
+        width: 24.9rem;
+        height: 4.1rem;
+        border-radius: 1.5rem;
+        border: 0.3rem solid var(--input-border);
+        margin: 0 2.1rem 1rem 2.1rem;
+    `}
+`;
+
```

Comment:
the last three lines are exact the same. Do you know, that styled-components have more than the option 'variant'? You can also receive props like width or height or or or. And again. I see much repetitions in all of your styled-component files. With the other props in mind, you should consider if this is the best solution. I think you can save a lot of work if you style the variants by passing props for width, height etc. Have another look at Domimik.